### DISCUSSION TITLE
Product Opportunity Explorer via SPAPI

### DISCUSSION THREAD

**Discussion by dorian-jellyfish** [amzn/selling-partner-api-models#3373] [General] [answered]
Hi,

I was wondering if there was any way of getting the metrics from the "Product Opportunity Explorer" section of the Seller Central UI, but with an SPAPI endpoint ? The documentation doesn't seem to indicate so, if this is well the case, would you consider implementing this in the future ?

Kind regards

**Comment by gravitiq-cm** [comment]
Would be great if Amazon sometimes replied to these posts, even to say "it's not on the roadmap" :-)

**Reply by maverick-marc** [reply]
It would also be helpful for me to get the Product Opportunity Explorer metrics via the api

**Reply by BrunnoPigatto** [reply, marked as answer]
Hello @dorian-jellyfish, @gravitiq-cm  and @maverick-marc,

Thank you for reaching out requesting the Product Opportunity Explorer feature.
Please note that this feature is already available in our [Customer Feedback v2024-06-01](https://developer-docs.amazon.com/sp-api/reference/customer-feedback-v2024-06-01) API, as can be seen [here](https://developer-docs.amazon.com/sp-api/changelog/new-introducing-the-customer-feedback-api-v2024-06-01).

Also, through the operation [searchCatalogItems](https://developer-docs.amazon.com/sp-api/reference/searchcatalogitems) of the [Catalog Items API v2022-04-01](https://developer-docs.amazon.com/sp-api/docs/catalog-items-api-v2022-04-01-use-case-guide), you can include `salesRanks` in the `summaries` query parameter to retrieve the Sales ranks of an Amazon catalog item.

Thanks,
Brunno M.
Selling Partner API Developer Service

**Reply by maverick-marc** [reply]
Thank you for sharing that link.  I see that [Customer Feedback v2024-06-01](https://developer-docs.amazon.com/sp-api/reference/customer-feedback-v2024-06-01) API is specific to the rating and review data within Product Opportunity Explorer.  The data I would like to be able to obtain includes the search terms associated with a niche and their metrics in addition to the products (asins) associated with a niche and their metrics as well.  Is this scheduled to be released anytime in the future? It would help me with monitoring metrics for a category that I and my clients are selling in as well as researching other potential niches that our clients can launch additional products in. Thank you!

**Comment by BrunnoPigatto** [comment]
Hello @maverick-marc,

Thank you for proposing an enhancement to the [Customer Feedback v2024-06-01](https://developer-docs.amazon.com/sp-api/reference/customer-feedback-v2024-06-01).
We submitted the feature request for review. Once we receive additional information we will post an update.

Thanks,
Brunno M.
Selling Partner API Developer Services
